docs/developer/architecture/core/uisettings-service.asciidoc

```diff
@@ -3,14 +3,68 @@
 
 NOTE: The UI settings service is available both server and client side.
 
-=== Server side usage
+=== Overview
+
+UI settings are configurable from the Advanced Settings page in Management and control the behavior of {kib}. `uiSettings` are stored in a `config` saved object and, as such, conform to the same conditions as other saved objects.
```

Comment:
> conform to the same conditions as other saved objects

Can we highlight the limitations/conditions that may affect this use case of SOs? Is it space-scoped? Migrations? Anything else?

Reply:
I've linked the text to the saved objects service.